dhcpv6/dhcpv6message.go

```diff
@@ -179,16 +179,40 @@ func (mo MessageOptions) UserClasses() [][]byte {
 	return nil
 }
 
-// VendorOpts returns the enterprise number and a list of vendor options.
-func (mo MessageOptions) VendorOpts() (uint32, Options) {
-	opt := mo.Options.GetOne(OptionVendorOpts)
+// VendorOpts returns the all vendor-specific options.
+//
+// RFC 8415 Section 21.17:
+//
+//   Multiple instances of the Vendor-specific Information option may appear in
+//   a DHCP message.
+func (mo MessageOptions) VendorOpts() []*OptVendorOpts {
```

Comment:
Why are you not returning the `EnterpriseNumber`?

Reply:
I am, though? OptVendorOpts includes the Enterprise num?

Reply:
Oh, right.